samples/nrf9160/fmfu_smp_svr/src/main.c

```diff
@@ -35,14 +67,16 @@ void main(void)
 		}
 
 	}
-	/* Shutdown modem to prepare for DFU */
-	nrf_modem_lib_shutdown();
-
-	nrf_modem_lib_init(FULL_DFU_MODE);
 	/* Register SMP Communication stats */
 	fmfu_mgmt_stat_init();
+#ifdef CONFIG_MCUMGR_CMD_OS_MGMT
+	os_mgmt_register_group();
```

Comment:
How are these relevant to this PR?

Reply:
Supposed to split up in to commits where this is just to add app update to the sample.

Reply:
Ah, my bad :)